toolchain/check/import_cpp.cpp

```diff
@@ -197,6 +199,76 @@ static auto ClangLookup(Context& context, SemIR::LocId loc_id,
   return lookup;
 }
 
+// Creates an integer type of the given size.
+static auto MakeIntType(Context& context, IntId size_id) -> TypeExpr {
+  // TODO: Fill in a location for the type once available.
+  auto type_inst_id = MakeIntTypeLiteral(context, Parse::NodeId::None,
+                                         SemIR::IntKind::Signed, size_id);
+  return ExprAsType(context, Parse::NodeId::None, type_inst_id);
+}
+
+// Maps a C++ type to a Carbon type.
+// Currently only int 32 is supported.
```

Comment:
I assume "32" here is the number of bits, but maybe that can be clarified in the comment?
```suggestion
// Currently only 32-bit `int` is supported.
```

Reply:
Done.